src/shared/ui/image/utils/checkSvg.ts

```diff
@@ -0,0 +1,8 @@
+const checkSvg = (imgUrl: string) => {
+  if (imgUrl.endsWith('.svg')) {
+    return true;
+  }
+  return false;
+};
+
+export default checkSvg;
```

Comment:
Remove default export

Reply:
![utils](https://github.com/rolling-scopes/site/assets/52886210/20232a7b-03f0-46a9-873d-156cd3db346d)
This entire section of code has a default export from me in one file.
if I remove default export, I will break the consistency.
Please, explain what benefits we will get if I do this

Reply:
In our team, we have decided to use only named exports. All code using default exports will be changed during the refactoring.

Reply:
Done